deeplake/core/index_maintenance.py

```diff
@@ -41,6 +41,18 @@ def validate_embedding_tensor(tensor):
     )
 
 
+def validate_text_tensor(tensor):
+    """Check if a tensor is an embedding tensor."""
+
+    valid_names = ["text", "id", "metadata"]
```

Comment:
do we want to force the users to use strictly defined tensor names for specific tensors? Do you know if we added in documentation appropriate information?

Reply:
Removed.